aidb/engine/full_scan_engine.py

```diff
@@ -3,15 +3,21 @@
 from typing import List
 
 from aidb.engine.base_engine import BaseEngine
-from aidb.inference.bound_inference_service import BoundInferenceService
 from aidb.query.query import Query
 
+QUERY_LIMIT = 10000
```

Comment:
What's this?

Reply:
To avoid extracting large volumes of data from the database. When utilizing a User Defined Function for a join condition, it requires performing a Cartesian product between the two tables, which may cause large memory usage if the entire resulting dataset is retrieved at once.

Reply:
This is for batching in that case, should be renamed